### PR TITLE
LUIS Runtime V3.0

### PR DESCRIPTION
**Pull request by nebadr**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md [hunk 1]
```diff
@@ -34,3 +34,12 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 ``` yaml $(tag) == 'runtime_3_0_preview' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/cognitiveservices/v3.0/luis/$(namespace)
 ```
+
+### Tag: runtime_3_0 and go
```
Comment (moabba) [inline specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: Remove the preview references before adding this.

File: specification/cognitiveservices/data-plane/LUIS/Runtime/readme.md [hunk 2]
```diff
@@ -37,6 +37,18 @@ directive:
   - remove-operation: Prediction_GetVersionPredictionGET
 ```
 
+## Runtime 3.0
+These settings apply only when `--tag=runtime_3_0` is specified on the command line.
+
+``` yaml $(tag) == 'runtime_3_0'
```
Comment (moabba) [inline specification/cognitiveservices/data-plane/LUIS/Runtime/readme.md]: Update the tag on line 8 to be `runtime_3_0`

File: specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v3.0/LUIS-Runtime.json [hunk 3]
```diff
@@ -0,0 +1,569 @@
...
+        },
+        "intents": {
+          "description": "A dictionary representing the intents that fired.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/Intent"
+          }
+        },
+        "entities": {
+          "description": "Gets or sets the dictionary representing the entities that fired.",
```
Comment (moabba) [inline specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v3.0/LUIS-Runtime.json]: Remove the "Gets or sets" prefix.

File: specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v3.0/LUIS-Runtime.json [hunk 4]
```diff
@@ -0,0 +1,569 @@
...
+    "ErrorBody": {
+      "description": "Represents the definition of the error that occurred.",
+      "required": [
+        "code",
+        "message"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Gets or sets the error code.",
```
Comment (moabba) [inline specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v3.0/LUIS-Runtime.json]: Remove the "Gets or sets" prefix.

File: specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v3.0/LUIS-Runtime.json [hunk 5]
```diff
@@ -0,0 +1,569 @@
...
+          "type": "string"
+        },
+        "message": {
+          "description": "Gets or sets the error message.",
```
Comment (moabba) [inline specification/cognitiveservices/data-plane/LUIS/Runtime/stable/v3.0/LUIS-Runtime.json]: Remove the "Gets or sets" prefix.

File: specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md (hunk 1 above)
Comment (nebadr) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: @moabba you mean all the preview here? or do i just fix the 'preview' in the v3.0 part?

Comment (moabba) [reply specification/cognitiveservices/data-plane/LUIS/Runtime/readme.go.md]: Everywhere in this files and the other files. Also delete the preview folder altogether.

**Review by moabba — APPROVED** [review]

**Review by anuchandy — APPROVED** [review]

**Pull request merged**